http-client-tls/Network/HTTP/Client/TLS.hs

```diff
@@ -122,7 +134,9 @@ convertConnection conn = makeConnection
 
 -- | Evil global manager, to make life easier for the common use case
 globalManager :: IORef Manager
-globalManager = unsafePerformIO (newManager tlsManagerSettings >>= newIORef)
+globalManager = unsafePerformIO $ do
+    man <- newManager $ mkManagerSettingsContext (Just globalContext) def Nothing
```

Comment:
Ah yes I was wondering. I compulsively use separate commits for such separate steps (so the choice reduces to cherry-picking), but I totally realize that's usually overkill 😅